CITATION.cff

```diff
@@ -0,0 +1,16 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Bezanson"
+  given-names: "Jeff"
+- family-names: "Edelman"
+  given-names: "Alan"
+- family-names: "Karpinski"
+  given-names: "Stefan"
+- family-names: "Shah"
+  given-names: "Viral B"
+title: "Julia: A fresh approach to numerical computing"
+version: 1.6.2
```

Comment:
```suggestion
version: 1.8.0-DEV
```

This value should be the same as the value in the `VERSION` file: https://github.com/JuliaLang/julia/blob/master/VERSION

Reply:
Also, we'll need to then change it for releases and such as well. What if we just don't include a version?

Reply:
See @storopoli's comment below - apparently GitHub will consider it invalid without the field.

Reply:
> Also, we'll need to then change it for releases and such as well.

Yeah it's going to get really annoying really quickly.

Could we just say `version: 1`? It's technically correct. And then we don't need to update it for a while 😂

Reply:
Or `version: v1`, or something.

Reply:
@arfon can we trick the version field like that?